providers/openfeature-go-feature-flag-provider/lib/openfeature/go-feature-flag/go_feature_flag_provider.rb

```diff
@@ -35,6 +35,12 @@ def evaluate(flag_key:, default_value:, allowed_classes:, evaluation_context: ni
         evaluation_context = OpenFeature::SDK::EvaluationContext.new if evaluation_context.nil?
         validate_parameters(flag_key, evaluation_context)
 
+        # enrich the evaluation context with the exporter_metadata
+        @options.exporter_metadata = {} if @options.exporter_metadata.nil?
+        @options.exporter_metadata["openfeature"] = true
+        @options.exporter_metadata["provider"] = "ruby"
```

Comment:
Those are default `exporter_metadata` options that are added for every evaluation.